x-pack/solutions/security/plugins/elastic_assistant/server/routes/attack_discovery/schedules/enable.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { IKibanaResponse, IRouter, Logger } from '@kbn/core/server';
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { buildRouteValidationWithZod } from '@kbn/zod-helpers';
+
+import {
+  API_VERSIONS,
+  EnableAttackDiscoverySchedulesRequestParams,
+  EnableAttackDiscoverySchedulesResponse,
+} from '@kbn/elastic-assistant-common';
+import { buildResponse } from '../../../lib/build_response';
+import { ATTACK_DISCOVERY_SCHEDULES_BY_ID_ENABLE } from '../../../../common/constants';
+import { ElasticAssistantRequestHandlerContext } from '../../../types';
+import { performChecks } from '../../helpers';
+
+export const enableAttackDiscoverySchedulesRoute = (
+  router: IRouter<ElasticAssistantRequestHandlerContext>
+): void => {
+  router.versioned
+    .put({
```

Comment:
Consider `put` -> `post`, because this route has side effects

Reply:
done in https://github.com/elastic/kibana/pull/216656/commits/d167ab0c749161c3c9e22fd949d1af479da97bc0